CONTRIBUTING.md

```diff
@@ -450,7 +452,8 @@ Before a PR can be merged, it should satisfy the following:
      - [ ] The miniapps go at the end of the page, and are usually listed only under a specific "Application (PDE)" category.
      - [ ] Add a short description of the miniapp in the "Extensive Examples" section of `features.md`.
 - [ ] New capability:
-   - [ ] All significant new classes, methods and functions have Doxygen-style documentation in source comments.
+   - [ ] All new public classes, methods and functions have full Doxygen-style documentation in source comments. Documentation should include descriptions of member data, function arguments and return values, and prerequisites for calling new functions.
```

Comment:
```suggestion
   - [ ] All new public, protected, and private classes, methods, data members and functions have full Doxygen-style documentation in source comments. Documentation should include descriptions of member data, function arguments and return values, template parameters, and prerequisites for calling new functions.
```

Reply:
I added this manually with one more Oxford comma